pipelines/rhtas-operator-e2e.yaml

```diff
@@ -0,0 +1,260 @@
+            emptyDir: {}
+          - name: repository
+            emptyDir: {}
+          - name: binaries
+            emptyDir: {}
+        steps:
+          - name: get-kubeconfig
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/build-definitions.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml
+            params:
+              - name: eaasSpaceSecretRef
+                value: $(tasks.provision-eaas-space.results.secretRef)
+              - name: clusterName
+                value: "$(tasks.provision-cluster.results.clusterName)"
+              - name: credentials
+                value: credentials
+          - name: run-operator
+            image: quay.io/operator-framework/operator-sdk:latest
+            env:
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+            args:
+              - run
+              - bundle
+              - --namespace
+              - "$(params.namespace)"
+              - --timeout
+              - "10m"
+              - "$(params.bundleImage)"
+          - name: git-clone
+            image: registry.access.redhat.com/ubi9/go-toolset@sha256:c7bfd2501cb1be171366434a368db669b32f08a0198c1473b9bff0a379613fc3
+            volumeMounts:
+              - name: repository
+                mountPath: /repository
+            script: |
+              git clone https://github.com/securesign/secure-sign-operator.git /repository
+          - name: install-keykloak
+            image: registry.redhat.io/openshift4/ose-cli
+            env:
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+              - name: repository
+                mountPath: /repository
+            script: |
+              cd /repository
+              ci/openshift/tas-keycloak-install.sh
+              oc get route -n keycloak-system keycloak -o jsonpath='{.status.ingress[0].host}' > /repository/oidc_host
+          - name: get-tuftool
+            image: registry.redhat.io/rhtas/tuftool-rhel9:1.1.0-1730740139
+            volumeMounts:
+              - name: binaries
+                mountPath: /binaries
+            script: cp /usr/bin/tuftool /binaries
+          - name: get-cosign
+            image: registry.redhat.io/rhtas/cosign-rhel9:1.1.0-1730377925
+            volumeMounts:
+              - name: binaries
+                mountPath: /binaries
+            script: cp /usr/local/bin/cosign /binaries
```

Comment:
Can this be separated out into a task maybe a "setup" task or something like that, to me this wouldn't be part of the e2e but instead the setup?

Reply:
I would like to but there is a problem with missing workspace concept in Pipeline (see https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1732705917913559). I am not aware about any way on how to pass `volume` to task but the [stepActions](https://tekton.dev/docs/pipelines/stepactions/) concept seems to be promising. WDYT?

Reply:
Yea the workspace issue is definitely a blocker, stepActions seems like the best option to use here